app/jobs/resolution_proofing_job.rb

```diff
@@ -39,22 +48,48 @@ def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
                           )
                         end
 
+    add_threatmetrix_result_to_callback_result(callback_log_data.result, threatmetrix_result)
+
     document_capture_session = DocumentCaptureSession.new(result_id: result_id)
     document_capture_session.store_proofing_result(callback_log_data.result)
   ensure
-    logger.info(
-      {
-        name: 'ProofResolution',
-        trace_id: trace_id,
-        resolution_success: callback_log_data&.resolution_success,
-        state_id_success: callback_log_data&.state_id_success,
-        timing: timer.results,
-      }.to_json,
+    logger_info_hash(
+      name: 'ProofResolution',
+      trace_id: trace_id,
+      resolution_success: callback_log_data&.resolution_success,
+      state_id_success: callback_log_data&.state_id_success,
+      timing: timer.results,
     )
   end
 
   private
 
+  def log_threatmetrix_info(threatmetrix_result, user)
+    logger_info_hash(
+      name: 'ThreatMetrix',
+      user_id: user&.uuid,
+      threatmetrix_request_id: threatmetrix_result.transaction_id,
+      threatmetrix_success: threatmetrix_result.success?,
+    )
+  end
+
+  def logger_info_hash(hash)
+    logger.info(hash.to_json)
+  end
```

Comment:
I am not convinced this helper method is worth it?

Reply:
was difficult to test multiple logger calls.  this made it much more simple to test

Reply:
it's also a tad bit DRYer